.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "tabWidth": 4,
+    "printWidth": 120
```

Comment:
I'm not sure about having a print width of `120` - the prettier docs specifically mention why the default of 80 makes sense. @giladgray do you have any comments here? Maybe we could set it to `100` so its in the middle.

Reply:
@johnwiseheart @giladgray are there any news?

Reply:
IMO 100 is a good choice because you can see the changes in a commit side by side without scrolling (on a 1080p monitor)

Reply:
I’ve always worked with 120, but 100 is a good compromise.

Also, the point the commit changes without scrolling is a good point.